django/api/models.py

```diff
@@ -1,3 +1,40 @@
 from django.db import models
 
-# Create your models here.
+class User(models.Model):
+    id = models.AutoField(primary_key=True)
```

Comment:
This here is not necessary as it is created automatically from django due to the fact that you extend the class models.Model :)